_data/tool_and_resource_list.yml

```diff
@@ -1290,3 +1290,407 @@
   id: camflow
   name: camflow
   url: https://camflow.org/
+- description: The R package will perform data quality checks against an OMOP CDM instance.
+  id: ohdsi
+  name: OHDSI
+  url: https://github.com/OHDSI/DataQualityDashboard
+- description: Tool to classify the taxonomic origin of a read in pathogen sequencing data
+  id: centrifuge
+  name: Centrifuge
+  url: https://bio.tools/centrifuge
+- description: R package for cell-type-specific co-expression inference from single cell RNA-sequencing data.
+  id: cs-score
+  name: CS_Score
+  url: https://github.com/ChangSuBiostats/CS-CORE
+- description: Tool to remove human reads from pathogen sequencing data.
+  id: deconseq
+  name: DeconSeq
+  url: https://bio.tools/deconseq
```

Comment:
Don't link to bio.tools but to the original page of the tool. Link to bio.tools tools in adition to the url using:
```
registry:
  biotools: deconseq
```

Chakc also all the other bio.tools urls